src/System.Management.Automation/engine/ProxyCommand.cs

```diff
@@ -9,19 +9,8 @@ namespace System.Management.Automation
     /// <summary>
     /// A ProxyCommand class used to represent a Command constructed Dynamically.
     /// </summary>
-    public sealed class ProxyCommand
+    public static class ProxyCommand
```

Comment:
Public API change. Local test failure: "IsAbstract value is not correct" but no failure on CI!

Reply:
reverted